cbor/codec.go

```diff
@@ -0,0 +1,29 @@
+package cbor
```

Comment:
The name of this package and the name of the cbor/v2 package are the same which is confusing perhaps